CONTRIBUTING.md

```diff
@@ -150,5 +150,24 @@ By making a contribution to this project, I certify that:
 ### Testing Your Code
 Please visit the [testing doc](tests/README.md) for details about how to run tests
 
+
+### Pre-commit hooks
+We provide a basic config `.pre-commit-config.yaml` for [pre-commit](https://pre-commit.com/) to
+automate some aspects of the development process. As a convenience you can enable automatic 
+copyright year updates by following the installation instructions on the
+[pre-commit homepage](https://pre-commit.com/), e.g.: 
+
+```bash
+conda install -c conda-forge pre-commit
```

Comment:
A little surprised to see `conda` here.  There's no discussion about setting up a conda environment anywhere in this repository nor is it required to have one.  `pre-commit` installs via `pip` and a bunch of other ways already covered on their homepage, so I was not expecting to see a specific one listed here which may not apply to someone's environment and then cause confusion when they attempt it.